main.css

```diff
@@ -0,0 +1,332 @@
+}
+
+.navabar__auth {
+  display: flex;
+  align-items: center;
+  gap: 2rem;
+}
+
+.navbar__login {
+  font-size: 0.8rem;
+  font-weight: 600;
+  line-height: 1.4rem;
+  cursor: pointer;
+  color: #1ba0ab;
+}
+
+.navbar__button {
+  display: flex;
+  align-items: center;
+  padding: 1rem 1.2rem;
+  gap: 0.5rem;
+  border: none;
+  font-size: 0.7rem;
+  font-weight: 600;
+  color: white;
+  border-radius: 5px;
+  cursor: pointer;
+}
+
+.navbar__button--primary {
+  background: #1ba0ab;
+}
+
+/* starting of showcase */
+
+.showcase {
+  background-color: #fff5f0;
+  padding: 1rem 0;
+}
+
+.showcase__container {
+  max-width: 82vw;
+  margin: 8.5rem auto;
+}
+
+.showcase__title {
+  font-size: 1rem;
+  font-weight: 600;
+  line-height: 1.5rem;
+  color: #1ba0ab;
+}
+
+.showcase__heading {
+  font-size: 3rem;
+  font-weight: 700;
+  line-height: 5rem;
+  letter-spacing: 0.2px;
+  color: #252b42;
+  margin-top: 1.4rem;
+  margin-bottom: 0.9rem;
+}
+
+.showcase__description {
+  font-size: 1.2rem;
+  font-weight: 500;
+  line-height: 1.8rem;
+  color: #737373;
+}
+
+.showcase__footer {
+  display: flex;
+  margin: 1.3rem 0;
+  gap: 0.5rem;
+}
+
+.showcase__button {
+  padding: 1rem 2rem;
+  border: 1px solid #1ba0ab;
+  font-size: 0.8rem;
+  font-weight: 700;
+  border-radius: 5px;
+  color: #1ba0ab;
+  cursor: pointer;
+}
+
+.showcase__button--primary {
+  color: white;
+  background-color: #1ba0ab;
+}
+
+/* starting of expert segment */
+
+.expert {
+  max-width: 70vw;
+  margin: 4rem auto;
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+}
+
+.expert__span {
+  width: 94px;
+  height: 7px;
+  background: #e74040;
+}
+.expert__title {
+  font-size: 2rem;
+  font-weight: 600;
+  line-height: 50px;
+  margin: 1.5rem 0;
+  color: #252b42;
+}
+
+.expert__description {
+  font-size: 0.8rem;
+  line-height: 1.25rem;
+  font-weight: 500;
+  color: #737373;
+}
+
+.expert__link {
+  margin-top: 1.5rem;
+  font-size: 0.8rem;
+  font-weight: 600;
+  line-height: 1.5rem;
+  color: #1ba0ab;
+  cursor: pointer;
+  display: flex;
+  gap: 0.5rem;
+  align-items: center;
+}
+
+/* starting of team segment */
+
+.team {
+  width: 73vw;
+  margin: 9rem auto;
+}
+
+.team__title {
+  font-size: 0.8rem;
+  font-weight: 600;
+  line-height: 1.5rem;
+  color: #1ba0ab;
+}
+
+.team__heading {
+  margin: 0.3rem 0;
+  font-size: 2rem;
+  font-weight: 600;
+  line-height: 50px;
+  color: #252b42;
+}
+
+.team__description {
+  width: 469px;
+  height: 40px;
+  font-size: 0.8rem;
+  line-height: 1.5rem;
+  color: #737373;
+}
+
+.team__card {
+  margin: 5rem 0;
+  display: flex;
+  gap: 1.5rem;
+}
+
+.card {
+  width: 250px;
+  height: auto;
+  background-color: white;
+  box-shadow: rgba(255, 255, 255, 0.1) 0px 1px 1px 0px inset,
+    rgba(50, 50, 93, 0.25) 0px 50px 100px -20px,
+    rgba(0, 0, 0, 0.3) 0px 30px 60px -30px;
+  border-radius: 1rem;
+  overflow: hidden;
+}
+
+.card__picture {
+  width: block;
+  height: 60%;
+}
+.card__picture img {
```

Comment:
this way of targeting element is not allowed in BEM, use only classes to target element